src/go/rpk/pkg/cli/debug/bundle/bundle.go

```diff
@@ -160,8 +161,8 @@ func NewCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 	f.StringVarP(&outFile, outputFlag, "o", "", "The file path where the debug file will be written (default ./<timestamp>-bundle.zip)")
 	f.DurationVar(&timeout, "timeout", 31*time.Second, "How long to wait for child commands to execute (e.g. 30s, 1.5m)")
 	f.DurationVar(&metricsInterval, "metrics-interval", 10*time.Second, "Interval between metrics snapshots (e.g. 30s, 1.5m)")
-	f.StringVar(&logsSince, "logs-since", "", "Include log entries on or newer than the specified date (journalctl date format, e.g. YYYY-MM-DD")
-	f.StringVar(&logsUntil, "logs-until", "", "Include log entries on or older than the specified date (journalctl date format, e.g. YYYY-MM-DD")
+	f.StringVar(&logsSince, "logs-since", "yesterday", "Include log entries on or newer than the specified date (journalctl date format, e.g. YYYY-MM-DD or today/yesterday")
```

Comment:
Got it, what do you think of:

```
Include logs dated from specified date onward; (journalctl date format: YYYY-MM-DD, 'yesterday', or 'today'). Refer to  journalctl documentation for more options.
```

To shorten it a little bit, this is a flag text and is already too large and it can be hard to read on small screens:
![image](https://github.com/redpanda-data/redpanda/assets/59714880/5194d666-c28b-4a4f-97d4-da401c8c876a)

Reply:
This is perfect!

Reply:
Thanks, fixed in https://github.com/redpanda-data/redpanda/compare/9396e476f542beeb0cd4ead7d2f0f17e268f349f..1b88856326f06d85b2f65db262bc99b2ce91c450